Misc/NEWS.d/next/Library/2018-10-24-17-43-21.bpo-33899.kBbX8r.rst

```diff
@@ -0,0 +1,2 @@
+Fix ``ntpath.abspath`` regression where it didn't remove a trailing
```

Comment:
The PR entry and NEWS entry appear to reference the wrong issue: bpo-33899 is a tokenizer issue.  It should be bpo-31047, no?  If so, please rerun blub and delete this NEWS file and fix the PR title.  Thanks!